PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -1,8 +1,11 @@
 import FWCore.ParameterSet.Config as cms
 
 from PhysicsTools.SelectorUtils.tools.vid_id_tools import *
+from Configuration.StandardSequences.Eras import eras
+
 
 def miniAOD_customizeCommon(process):
+    process.load("PhysicsTools.PatAlgos.slimming.era_customizers")
```

Comment:
you can at least put miniAOD_forPhase2 in a separate file, maybe in the era_customizers itself
right now from file A (this file) you load file B (era_customizers), which loads elements of A